addons/website_sale_wishlist/views/website_sale_wishlist_template.xml

```diff
@@ -18,7 +18,7 @@
         <xpath expr="//div[hasclass('o_wsale_product_btn')]" position="inside">
             <t t-set="in_wish" t-value="product._is_in_wishlist()"/>
             <t t-set="product_variant_id" t-value="product._get_first_possible_variant_id()"/>
-            <button t-if="product_variant_id" type="button" role="button" class="btn btn-secondary o_add_wishlist" t-att-disabled='in_wish or None' title="Add to Wishlist" t-att-data-product-template-id="product.id" t-att-data-product-product-id="product_variant_id" data-action="o_wishlist"><span class="fa fa-heart" role="img" aria-label="Add to wishlist"></span></button>
```

Comment:
Unrelated-to-the-line comment:
![Screenshot from 2021-03-29 16-38-13](https://user-images.githubusercontent.com/10338094/112853564-449fd700-90ad-11eb-85ff-2a3fb41e1f93.png)

- Should not the heart be on the far right instead of between the two buttons?
- Should not the heart be filled when it has been favorited?

Reply:
ok :+1: